app/src/main/java/com/framgia/fsalon/screen/homeadmin/AdminHomeViewModel.java

```diff
@@ -35,48 +36,46 @@
     private HomePagerAdapter mAdapter;
     private int mTabPosition;
     private BottomNavigationView.OnNavigationItemSelectedListener mListener =
-        new BottomNavigationView.OnNavigationItemSelectedListener() {
-            @Override
-            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-                switch (item.getItemId()) {
-                    case R.id.item_scheduler:
-                        setTabPosition(Tab.TAB_SCHEDULER);
-                        break;
-                    case R.id.item_receipt:
-                        /**
-                         * todo later
-                         */
-                        setTabPosition(Tab.TAB_RECEIPT);
-                        break;
-                    case R.id.item_report:
-                        /**
-                         * todo later
-                         */
-                        setTabPosition(Tab.TAB_REPORT);
-                        break;
-                    case R.id.item_customer:
-                        /**
-                         * todo later
-                         */
-                        setTabPosition(Tab.TAB_CUSTOMER);
-                        break;
-                    case R.id.item_profile:
-                        /**
-                         * todo later
-                         */
-                        setTabPosition(Tab.TAB_PROFILE);
-                        break;
+            new BottomNavigationView.OnNavigationItemSelectedListener() {
+                @Override
+                public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                    switch (item.getItemId()) {
+                        case R.id.item_scheduler:
+                            setTabPosition(Tab.TAB_SCHEDULER);
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'Tab.TAB_SCHEDULER' due to existing static import 'com.framgia.fsalon.screen.homeadmin.AdminHomeViewModel.Tab.TAB_SCHEDULER'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName